cpg_utils/config-template.toml

```diff
@@ -150,7 +150,7 @@ dry_run = false
 # Docker image URLs. Can be absolute or relative to `workflow.image_registry_prefix`.
 gatk = 'gatk:4.2.6.1'
 bcftools = 'bcftools:1.10.2--h4f4756c_2'
-sm-api = 'sm-api:4.0.0'
+sm-api = 'australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:7d00c4871b2e96f50bae208e4184c3c4789a2fa4-hail-83056327f288917537531475ba475287b413db1c'
```

Comment:
Do you have to update did everytime the driver image updates? Feels very manual

Reply:
Yep, like other image versions. Not sure what's the best approach here